pkg/hub/controllers/cluster/reconciler.go

```diff
@@ -72,6 +73,7 @@ func (r *Reconciler) Reconcile(ctx context.Context, req reconcile.Request) (reco
 	cr.Status.ClusterHealth.LastUpdated = metav1.Now()
 	if err := r.Status().Update(ctx, cr); err != nil {
 		log.Error(err, "unable to update cluster CR")
+		return reconcile.Result{RequeueAfter: ReconcileInterval}, err
```

Comment:
This should be return reconcile.Result{Requeue: true}, err, why returning after interval?